templates/beta-release-post.adoc

```diff
@@ -117,7 +117,7 @@ If you're using link:{url-prefix}/guides/maven-intro.html[Maven], you can instal
 <plugin>
     <groupId>io.openliberty.tools</groupId>
     <artifactId>liberty-maven-plugin</artifactId>
-    <version>RELEASE</version>
+    <version>3.8.2</version>
```

Comment:
Does `RELEASE_VERSION` in the `runtimeArtifact` get replaced by the actual version when you build the blog post?

Reply:
yes; it gets replaced with the Liberty version for which the blog post is being created (e.g. 23.0.0.10 or 23.0.0.10-beta)